prime-router/docs/api/components/security_schemes.yml

```diff
@@ -0,0 +1,14 @@
+OktaAuth:
+  type: oauth2
+  flows:
+    authorizationCode:
+      authorizationUrl: https://hhs-prime.okta.com/oauth/authorize
+      tokenUrl: https://hhs-prime.okta.com/oauth/token
+      scopes:
+        ANONYMOUS: Grants read access
+        FUNCTION: Grants write access to single org
```

Comment:
"ANONYMOUS" and "FUNCTION" are not clear names.

The older specs use:
          scopes:
            user: Grants read access
            org_admin: Grants write access to single org
            system_admin: Grants access to admin operations

which is a little more descriptive.

Reply:
Sounds good, updated!